src/Parser/AfflictionWarlock/Modules/SoulShards/SoulShardDetails.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import SoulShardBreakdown from './SoulShardBreakdown';
+import SoulShardTracker from './SoulShardTracker';
+
+import WastedShardsIcon from '../../Images/affliction_warlock_soulshard_bw.jpg';
+
+const soulShardIcon = 'inv_misc_gem_amethyst_02';
+
+class SoulShardDetails extends Module {
+  static dependencies = {
+    soulShardTracker: SoulShardTracker,
+  };
+
+  suggestions(when) {
+    const shardsWasted = this.soulShardTracker.shardsWasted;
+    when(shardsWasted).isGreaterThan(0)
```

Comment:
No soul shards wasted is an ideal situation in any fight, but it depends on the fight a lot of time. Not in the duration but in the kind of the fight (for example, on Kil'jaeden I have around 20 wasted shards because I kept Agony up on KJ in the flying phase, just to keep it from falling off, but it kept generating shards). Not sure what number is appropriate to give as a recommended amount, because technically it should be as close to zero as possible.